lib/std/crypto/tls/Client.zig

```diff
@@ -295,14 +299,27 @@ pub fn init(stream: anytype, ca_bundle: Certificate.Bundle, host: []const u8) In
                             const key_size = extd.decode(u16);
                             try extd.ensure(key_size);
                             switch (named_group) {
+                                .x25519_kyber768d00 => {
+                                    const ksl = 32 + crypto.kem.kyber_d00.Kyber768.ciphertext_length;
+                                    if (key_size != ksl)
+                                        return error.TlsIllegalParameter;
+                                    const server_ks = extd.array(ksl);
+
+                                    shared_key = &((crypto.dh.X25519.scalarmult(
+                                        x25519_kp.secret_key,
+                                        server_ks[0..32].*,
+                                    ) catch return error.TlsDecryptFailure) ++ (kyber768_kp.secret_key.decaps(
+                                        server_ks[32..ksl],
```

Comment:
The hardcoded `32` for X25519 could be replaced with `dh.X25519.public_length` for consistency and clarity.
(same in the `.x25519` block below)

Reply:
Done.